wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
It seems like our pattern so far is to return BadRequest when an action is not valid. If I were to use some kind of static constructor, I would have to return something instead of the created object to make it clear that the action attempted was not valid. It might be more intuitive to have a method like `CanConstructFromDto` that returns a bool and a separate call to the constructor itself so that it is explicit to the user when it is acceptable to create an entity or not.